src/components/Editor.jsx

```diff
@@ -71,6 +71,9 @@ class Editor extends React.Component {
       requestedFocusedLine.line,
       requestedFocusedLine.column
     );
+
+    this._scrollToLine(requestedFocusedLine.line);
```

Comment:
I decided to keep this in the focus line method/action rather than creating a separate one because it seems like expected behavior to scroll to the line when focusing the cursor